test/test-lib-nodejs.sh

```diff
@@ -549,18 +549,31 @@ function run_all_tests() {
 # Check the imagestream
 function test_nodejs_imagestream() {
   case ${OS} in
-    rhel7|centos7) ;;
+    rhel7|centos7|rhel8|rhel9) ;;
     *) echo "Imagestream testing not supported for $OS environment." ; return 0 ;;
   esac
 
+  local tag="-ubi7"
+  if [ "${OS}" == "rhel8" ]; then
+    tag="-ubi8"
+  elif [ "${OS}" == "rhel9" ]; then
+    tag="-ubi9"
+  fi
+  if [ "${VERSION}" == "14-minimal" ]; then
+    VERSION=14
```

Comment:
This could also be done without the conditions, with some regex. That would be better I think.

Reply:
REsolved by https://github.com/sclorg/s2i-nodejs-container/pull/364/commits/194da2603db785ee6b7e6fd7ad81f7e294650a29